app/views/layouts/_header.html.erb

```diff
@@ -11,15 +11,12 @@
     <div class="container">
       <div class="row">
         <div class="col-md-2 col-sm-3 col-xs-3">
-          <%= link_to image_tag('logo.jpg', size: '50x50'), rails_admin_path %>
+          <%= link_to image_tag('logo.jpg', size: '50x50'), root_path %>
         </div>
         <div class="col-md-10 col-sm-9 col-xs-9 spa-algr" style="height: 100px;">
           <nav class="spa-menu-top">
             <ul>
               <li>
-                <%= link_to "Home", root_path %>
-              </li>
-              <li>
```

Comment:
this link is good, we should not remove it, not all users know to click the logo to go to root, if you want we nac change it to "Strona główna"